### PR TITLE
Add v4.5 reference assemblies

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@jaredpar

**Comment by tmat** [comment]
@jaredpar Ping

File: build-reference-assemblies.ps1 [hunk 1]
```diff
@@ -17,13 +17,14 @@ function Print-Usage() {
...
 
-# TODO: remove this step once we have a valid framework asesmbly project
+# TODO: remove this step once we have a valid framework assembly project
 function Compose-Framework() {
     Write-Host "Composing reference assemblies"
     $copyList = @(
         ".NETCore\v5.0",
         ".NETFramework\v4.0",
         ".NETFramework\v4.6",
+        ".NETFramework\v4.5",
```
Comment (jaredpar) [inline build-reference-assemblies.ps1]: Can you add this above 4.6 to keep the sorting?

**Comment by jaredpar** [comment]
Have you rebuilt this package yet? If not I can rebuild it and push.

**Comment by tmat** [comment]
Haven't rebuilt it yet.

**Comment by jaredpar** [comment]
Rebuilding now.

**Comment by tmat** [comment]
Thanks

**Comment by jaredpar** [comment]
@tmat version 0.1.3 uploaded. Prob take a few for myget to start listing it.

**Pull request merged**